apiv6/examples/synthetics/main.go

```diff
@@ -0,0 +1,522 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/kentik/community_sdk_golang/apiv6/examples"
+	"github.com/kentik/community_sdk_golang/apiv6/kentikapi"
+	"github.com/kentik/community_sdk_golang/apiv6/kentikapi/synthetics"
+)
+
+// Current status:
+// CreateAgent                     POST   - missing request payload data struct in swagger spec
```

Comment:
Hmm, It might be the case that you are looking at some older revision of the code, the recent revision is Apr 22, 2021 "Added support for synthetics in localhost_apiserver" (I did a correcting overriding git push there, seems this introduced confusion; will avoid doing this again). Current status:
- generated docs removed
- spec files removed
- recent spec from public repo applied (PATCH works)
- examples for synthetics work (except for CREATE tests)